napari/components/viewer_model.py

```diff
@@ -402,19 +401,6 @@ def _update_cursor_size(self, event):
         """Set the viewer cursor_size with the `event.cursor_size` int."""
         self.cursor.size = event.cursor_size
 
-    def _on_cursor_position_change(self):
```

Comment:
This is unrelated, but I figure we could get rid of this other callback too. `Layer.position` was [deprecated a while ago](https://github.com/napari/napari/pull/2443), though I'm not actually sure when it was removed.